snuba/api.py

```diff
@@ -5,6 +5,7 @@
 from datetime import datetime, timedelta
 from flask import Flask, render_template, request
 from markdown import markdown
+from uuid import uuid4
```

Comment:
Let's make this `uuid1` instead of `uuid4` so we can extract the time component if we need to.